WalletWasabi/BitcoinCore/Rpc/RpcParser.cs

```diff
@@ -27,53 +28,63 @@ public static RpcPubkeyType ConvertPubkeyType(string pubKeyType)
 
 		public static VerboseBlockInfo ParseVerboseBlockResponse(string getBlockResponse)
 		{
-			var blockInfoJson = JObject.Parse(getBlockResponse);
-			var previousBlockHash = blockInfoJson.Value<string>("previousblockhash");
+			var parsed = JsonDocument.Parse(getBlockResponse).RootElement;
+			if (!parsed.TryGetProperty("result", out JsonElement blockInfoJson))
+			{
+				blockInfoJson = parsed;
+			}
+
+			string? previousBlockHash = null;
+			if (blockInfoJson.TryGetProperty("previousblockhash", out var prevBlockHashJson))
+			{
+				previousBlockHash = prevBlockHashJson.GetString();
+			}
+
 			var transaction = new List<VerboseTransactionInfo>();
```

Comment:
`var transaction = new List<VerboseTransactionInfo>();` should probably be called `transactions`.

Reply:
Done!